### ISSUE TITLE
Oneliner not for current conditions but for a few hours forecast?

### ISSUE THREAD

**Issue by 097115** [chubin/wttr.in#329]
Hi!

As I understand, one line reports, like `curl https://wttr.in/London?format=%l+%t+%h+%w` inform us about the **current** conditions. 

But may be it would be possible to add some switch, like `%f=1`, to tell what weather is expected to be in one hour's time?

Or, since my personal point is to check for the rain :), then may be some other switch would be possible, like `%r=3`, which would report like `[...], no rain is expected in 3 hours time` or something like that.

What do you think? :)

Thanks for the app!

**Comment by chubin** [comment]
Yes, it is a very useful feature. @dubl3a already mentioned it in #174 and I hope it will be available soon.
I will post more information about this feature here, as soon as it is available

**Comment by chubin** [comment]
@097115 No, you haven't miss anything, this feature is still not implemented, but it is pretty close to be implemented now. The main problem with this feature was that the server could not handle load which was too high even with additional one-line features (because it is one-line format who produces the biggest fraction of load). Now when the problem is fixed, we can accelerate with all one-line features, and first of all this one.

I hope that I will have something to show maybe even on this weekend

**Comment by mstr3336** [comment]
Has this been implemented yet?

**Comment by chubin** [comment]
Sorry guys, not yet, I am totally busy with other projects, but as soon as I am back, I will do it

**Comment by chubin** [comment]
(also see #447)

**Comment by shirishag75** [comment]
Whenever you put in the 3 day forecast for the one-line method, please share here too with an example how it will work so we know. Thank you.

**Comment by ACR-Jeff** [comment]
> Whenever you put in the 3 day forecast for the one-line method, please share here too with an example how it will work so we know. Thank you.

I agree, It's tough finding information on how to use this tool and examples. I can find some information on using wttr.in but examples and information are tough to find. Not many seems to explain much on Github. Very confusing.

**Comment by elsurudo** [comment]
Has this been implemented, or is it planned still? I have a need for it (specifically rain forecast), and I'm wondering if I can use wttr.in or if I should look further.

This project looks great otherwise!
